plugins/UFM_NDT_Plugin/ufm_sim_web_service/resources.py

```diff
@@ -57,6 +57,7 @@ def __init__(self):
         self.ndts_list_file = os.path.join(self.ndts_dir, "ndts_list.json")
         self.reports_merger_list_file = os.path.join(self.reports_merger_dir, "merger_reports_list.json")
         self.ndts_merger_list_file = os.path.join(self.ndts_merger_dir, "ndts_list.json")
+        self.cv_credentials_path = "/config/cv_credentials"
```

Comment:
should not the path of cable validation credentials be part of NDT configuration file?

Reply:
No. It is hardcoded.